docs/core/OTTER_ANGULAR_TOOLS.md

```diff
@@ -99,6 +99,7 @@ In angular.json file of your lib.
 | store-action        | Create an action into an existing store.                              | `ng generate @o3r/core:store-action`        |
 | storybook-component | Create an Storybook file for a presenter.                             | `ng generate @o3r/core:storybook-component` |
 | renovate-bot        | Create a basic Renovate Bot                                           | `ng generate @o3r/core:renovate-bot`        |
+| ng-add-create       | Include `ng add` schematic into your project                          | `ng generate @o3r/core:ng-add-create`       |
```

Comment:
More global comment but this list should not be part of the core package's readme.md instead of this file?

Reply:
true. we can put all the generators there.